hub/api/tests/test_update_samples.py

```diff
@@ -380,3 +380,34 @@ def test_byte_positions_encoder_update_bug(memory_ds):
     assert ds.abc[0].numpy().shape == (2, 2)
     for i in range(1, 10):
         assert ds.abc[i].numpy().shape == (1, 1)
+
+
+@pytest.mark.parametrize(
+    "args",
+    [
+        {},
+        {"sample_compression": "lz4"},
+        {"chunk_compression": "lz4"},
+        {"sample_compression": "png"},
+        {"chunk_compression": "png"},
+    ],
+)
+@pytest.mark.parametrize("htype", ["generic", "sequence"])
+def test_update_partial(memory_ds, htype, args):
```

Comment:
add a test for partially updating tiled samples (unless we already have it)

Reply:
We already have it.